maintainers/maintainer-list.nix

```diff
@@ -944,6 +944,11 @@
     githubId = 5718007;
     name = "Bastian Köcher";
   };
+  blanky0230 = {
+    email = "blanky0230@gmail.com";
+    github = "blanky0230";
```

Comment:
`githubId` missing, see top of this file

Reply:
Thanks! I wasn't aware that it now is required :)